### PR TITLE
Fix back links and markdown copy of qubit-guarantee

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: content/pieces/back/index.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ https://imagesnippets.com/
 ### b⃣ . e⃣ . n⃣ . n⃣ . y⃣ .
 ```
 https://www.instagram.com/twicetunatuna/
-https://newart.city
-https://gentleplants.com
+https://newart.city/
+https://gentleplants.com/
```
Comment (uditvira) [inline content/pieces/back/index.md:50]: lol I thought we agreed that inconsistencies == vibes

Comment (uditvira) [reply content/pieces/back/index.md:50]: @benhylau couldn't resist to sneak this in

Comment (benhylau) [reply content/pieces/back/index.md:50]: Oh I thought that didn't cross the bar for a hotfix, but if we are deploying anyways, we should fix, no?

Comment (uditvira) [reply content/pieces/back/index.md:50]: yeah just as always, impressed with your super attention to detail

**Review by uditvira — APPROVED** [review]

**Comment by uditvira** [comment]
- merged celine's issue into hotfix branch
- merging to master

**Pull request merged**